resources/wikia/modules/tracker.js

```diff
@@ -161,13 +165,13 @@
 
 			// Set up params object - this should stay in sync with /extensions/wikia/Track/Track.php
 			params = {
-				'c': window.wgCityId,
-				'x': window.wgDBname,
-				'a': window.wgArticleId,
-				'lc': window.wgContentLanguage,
-				'n': window.wgNamespaceNumber,
+				'c': window.wgCityId || (wikiVariables && wikiVariables.id),
+				'x': window.wgDBname || (wikiVariables && wikiVariables.dbName),
+				'a': window.wgArticleId || (trackingDimensions && trackingDimensions[21]),
```

Comment:
A question aside from this PR, do you have an idea on how to avoid using numeric indices here?

Reply:
Not really because key names are not saved to this object. You can check it here:
https://github.com/Wikia/app/blob/76632441e034e248ed0a5326bdd330410a143025/extensions/wikia/AnalyticsEngine/js/universal_analytics.js#L336